budget-service/pom.xml

```diff
@@ -16,13 +16,32 @@
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot</artifactId>
+        </dependency>
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-web</artifactId>
         </dependency>
+
         <dependency>
             <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot</artifactId>
+            <artifactId>spring-boot-starter-data-jdbc</artifactId>
+            <version>3.3.0</version>
```

Comment:
Can we define these dependencies in the parent pom's dependencyManagement and remove their version from this pom file?

Reply:
This still stands
This pom file shouldn't contain any versions.
The versions for all of these dependencies should be set in the parent's dependencyManagement